client/src/components/User/DiskUsage/Visualizations/Charts/utils.ts

```diff
@@ -0,0 +1,10 @@
+import { bytesToString } from "@/utils/utils";
+import type { DataValuePoint } from ".";
+
+export function bytesLabelFormatter(dataPoint?: DataValuePoint | null): string {
+    return dataPoint ? `${dataPoint.label}: ${bytesToString(dataPoint.value)}` : "No data";
+}
+
+export function bytesValueFormatter(value: number): string {
+    return bytesToString(value);
+}
```

Comment:
`formatters` or `formating` may be more fitting module names than `utils`.
I'd like to keep the amount of `util` modules in the project to a minimum, as a module with such a name could contain anything which tends to make code harder to logically follow.